nimterop.nimble

```diff
@@ -12,14 +12,21 @@ installDirs = @["nimterop"]
 requires "nim >= 0.20.2", "regex >= 0.14.1", "cligen >= 0.9.45"
 
 import nimterop/docs
+import os
 
 proc execCmd(cmd: string) =
   exec "tests/timeit " & cmd
 
-proc execTest(test: string, flags = "") =
+proc execTest(test: string, flags = "", runDocs = true) =
```

Comment:
Do you still need this? You fixed rsa.nim right?

Reply:
Probably don't need it, but thought I would leave it it just in case we want to disable docgen easily for some reason. I can remove it if you want